### ISSUE TITLE
mapview does not support vector mbtiles

### ISSUE THREAD

**Issue by RedFiveAZ** [kivy-garden/garden.mapview#22]
I will try my hand at making this work, but I'm not sure my skills are high enough for this. would love help...

**Comment by TimSC** [comment]
I don't think this supports vector tiles and it would be a big job to make it happen. Vector tile renderers:

https://github.com/mapbox/mapbox-gl-native
https://github.com/TimSC/iridescent-gtk3

(I originally thought project supported vector tiles but it actually only seems to support raster tiles. See https://github.com/kivy-garden/garden.mapview/pull/19 )

**Comment by RedFiveAZ** [comment]
Yes, I concur that it would be a big job! I was hoping that the author might pop on here and say he was currently working on it, but I suspect that I'll have to write my own solution!

**Comment by tito** [comment]
You're right, that would be just awesome!
